distribution-spec/v1.1/keygen/README.md

```diff
@@ -0,0 +1,43 @@
+# Keygen
+
+Keygen implements a read-only OCI distribution engine, for both public and
+licensed container images. Below are instructions on how to run the OCI
+conformance test against Keygen's API.
+
+## Running the test
+
+Clone OCI distribution spec repository:
+
+```
+git clone https://github.com/opencontainers/distribution-spec -b v1.1.0
+```
+
+Build conformance test binary:
+
+```sh
+cd distribution-spec/conformance
+
+go test -c
+```
+
+Set test environment variables:
+
+```sh
+export OCI_ROOT_URL=https://oci.pkg.keygen.sh
+export OCI_USERNAME=license
+export OCI_PASSWORD=9EBD41-A4BCBF-4667C5-286307-C29748-V3
```

Comment:
Just to verify, you are okay with this credential being publicly shared? If not, now that's it's public you would want to rotate it.

Reply:
Totally fine. Just a test account.